src/System.Management.Automation/engine/remoting/commands/StartJob.cs

```diff
@@ -623,24 +654,23 @@ protected override void CreateHelpersForSpecifiedComputerNames()
                             ErrorCategory.PermissionDenied,
                             Context.LanguageMode));
             }
-
+            
```

Comment:
Looks like there a handful of lines in this file you added a bit of whitespace that doesn't need to be here. Probably best to remove it so as not to complicate the diffs in any future edits. 🙂

Reply:
fixed in https://github.com/davinci26/PowerShell/commit/06ef81a76db4898ec1066f69e50558d47f8e85f0